aiida_common_workflows/workflows/relax/orca/generator.py

```diff
@@ -159,6 +160,11 @@ def get_builder( # pylint: disable=too-many-branches, too-many-statements
                 # round guess to nearest even integer; 0 goes to 2
                 spin_multiplicity = max([int(np.round(multiplicity_guess / 2) * 2), 2])
 
+            if spin_multiplicity == 1:
+                params['input_blocks']['scf']['STABPerform'] = True
+                if 'EnGrad' in params['input_keywords']:
+                    params['input_keywords'].remove('EnGrad')
```

Comment:
why remove the `engrad` for the open-shell singlet?

Reply:
It does not work with `STABPerform`. On the other hand, using the other option (`SpinFlip` and `FinalSpin`) can become tricky. I kept it this way for now, until we would have results for more complex systems (hopefully in future).

Reply:
okay, i see. I tested the flipspin a bit as well and indeed it's not as robust as i though